server/index.ts

```diff
@@ -0,0 +1,12 @@
+import express, { Application, Request, Response } from 'express';
+
+const app: Application = express();
+const port: number = 3000;
```

Comment:
I think this is good for the base for now, but at some point port number is preferably passed through an environmental variable.

Reply:
@Gskartwii has already implemented this at https://github.com/Gskartwii/aalto-grades-base/tree/backend-base